<ui version="4.0" >
 <class>runForm</class>
 <widget class="QWidget" name="runForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>415</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Run job</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="run" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>&lt;h3>Run a Job&lt;/h3></string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>5</height>
      </size>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_12" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../main.qrc" >:/images/runit.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Job properties</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>Job:</string>
          </property>
          <property name="buddy" >
           <cstring>jobCombo</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QComboBox" name="jobCombo" />
        </item>
        <item row="0" column="2" colspan="2" >
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>0</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string>Type:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="typeLabel" >
            <property name="text" >
             <string>&lt;h3>Backup&lt;h3/></string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_8" >
          <property name="text" >
           <string>Client:</string>
          </property>
          <property name="buddy" >
           <cstring>clientCombo</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QComboBox" name="clientCombo" />
        </item>
        <item row="1" column="2" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Priority:</string>
          </property>
          <property name="buddy" >
           <cstring>prioritySpin</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QSpinBox" name="prioritySpin" >
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="value" >
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_11" >
          <property name="text" >
           <string>Level:</string>
          </property>
          <property name="buddy" >
           <cstring>levelCombo</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QComboBox" name="levelCombo" />
        </item>
        <item rowspan="5" row="2" column="3" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>56</width>
            <height>151</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_9" >
          <property name="text" >
           <string>FileSet:</string>
          </property>
          <property name="buddy" >
           <cstring>filesetCombo</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QComboBox" name="filesetCombo" />
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Pool:</string>
          </property>
          <property name="buddy" >
           <cstring>poolCombo</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="QComboBox" name="poolCombo" />
        </item>
        <item row="5" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Storage:</string>
          </property>
          <property name="buddy" >
           <cstring>storageCombo</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1" >
         <widget class="QComboBox" name="storageCombo" />
        </item>
        <item row="6" column="0" >
         <widget class="QLabel" name="label_10" >
          <property name="text" >
           <string>Messages:</string>
          </property>
          <property name="buddy" >
           <cstring>messagesCombo</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="1" >
         <widget class="QComboBox" name="messagesCombo" />
        </item>
        <item row="7" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>When:</string>
          </property>
          <property name="buddy" >
           <cstring>dateTimeEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="7" column="1" >
         <widget class="QDateTimeEdit" name="dateTimeEdit" >
          <property name="dateTime" >
           <datetime>
            <hour>0</hour>
            <minute>2</minute>
            <second>0</second>
            <year>2000</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="displayFormat" >
           <string>yyyy-mm-dd hh:mm:ss</string>
          </property>
          <property name="calendarPopup" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="0" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Bootstrap:</string>
          </property>
          <property name="openExternalLinks" >
           <bool>true</bool>
          </property>
          <property name="buddy" >
           <cstring>bootstrap</cstring>
          </property>
         </widget>
        </item>
        <item row="8" column="1" >
         <widget class="QLineEdit" name="bootstrap" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="minimumSize" >
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="readOnly" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>5</height>
      </size>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../main.qrc" />
 </resources>
 <connections/>
</ui>
